docs/rackhd/api-usage-examples.rst

```diff
@@ -21,21 +21,26 @@ This example provides instructions on how to flash a BMC image on a Quanta (node
 2. Post the obm settings if they don't already exist for the node. An example on how to do this is shown in Section 7.1.8.1 here http://rackhd.readthedocs.io/en/latest/tutorials/vagrant.html#adding-a-sku-definition Section 7.1.8.1
 
 
-3. The firmware files need to be built into a SKU package
+3. Quanta-T41, Quanta-D51 Firmware upgrades:
+   Firmware upgrade process for these platforms is contained in the README.md file in their folders under on-skupack repositories.
+   Dell R730/R730XD, R630, C6320 Firmware upgrade(To be updated)
+
+4.  The firmware files need to be built into a SKU package
   
      **Build SKU Package**
      ::
      
-         $ ./build-package.bash <sku_pack_directory> <subname>
+         $ ./build-package.bash <sku_pack_name> <subname>
```

Comment:
https://github.com/RackHD/on-skupack 's readme has the same 'sku_pack_directory' parameter description. is it worth to update there?